client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/premier-agency-hosting/index.tsx

```diff
@@ -20,8 +23,37 @@ type Props = {
 export default function PremierAgencyHosting( { onAddToCart }: Props ) {
 	const translate = useTranslate();
 
+	const { pressablePlans } = useProductAndPlans( {
+		selectedSite: null,
+		productSearchQuery: '',
+	} );
+
+	const { existingPlan, isReady: isExistingPlanFetched } = useExistingPressablePlan( {
+		plans: pressablePlans,
+	} );
+
 	return (
 		<div className="premier-agency-hosting">
+			{ isExistingPlanFetched && existingPlan && (
```

Comment:
Should we make sure this does not show up in production while it is not 100% Ready?